.github/workflows/generate.yaml

```diff
@@ -0,0 +1,34 @@
+name: Update ts-proto library
+on:
+  repository_dispatch:
+    types: [protobuf_changes]
+jobs:
+  update-ts-proto:
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: ${{ github.event.client_payload.taskID }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.PAT }}
+
+      # Setup for pushing to Buf.build later
+      - uses: bufbuild/buf-setup-action@v1.9.0
+
+      - name: Generate TS code for protobufs
+        run: buf generate buf.build/cheqd/cheqd-proto
+      
+      - name: Push to origin
+        run: |
+          git checkout -B ${{ env.BRANCH }}
+          git add .
+          git config --global user.name 'AutomationUser'
+          git config --global user.email 'AutomationUser@users.noreply.github.com'
+          git commit -am "chore(deps): Update protobufs"
+          git push --force origin ${{ env.BRANCH }}
+
+      - name: Create Pull Request
+        run: gh pr create -B main -H ${{ ENV.BRANCH }} --title 'chore(deps)':' Update protobufs' --body 'Created by Github action' || true
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
```

Comment:
I'm unsure what's this for. `chore` especially seems incorrect since chore will NOT trigger a release. 

To push, we already have a buf push action. Unsure about this hardcoded PR creation?

Also, I don't understand the logic behind this buf generation since there's no buf.yaml file.